multibase.csv

```diff
@@ -15,3 +15,4 @@ base64,       m,    rfc4648 no padding
 base64pad,    M,    rfc4648 with padding - MIME encoding
 base64url,    u,    rfc4648 no padding
 base64urlpad, U,    rfc4648 with padding
+binary,       bB,   8-bit binary (encoder and decoder keeps data unmodified)
```

Comment:
This conflicts with base32.  The README is out of date.

Also it should only be a single symbol.  Bases with two symbols are by there nature case intensive for example in hex `2a3b` is the same as `2A3B`.